LICENSES/LicenseRef-Slint-Royalty-free-1.2.md

```diff
@@ -30,6 +30,8 @@ You may distribute the Software as part of an Application, modified or unmodifie
 
 The license does not permit to distribute or make the Software publicly available alone and without integration into an Application. For this purpose you may use the Software under the GNU General Public License, version 3.
 
+The distribution of the Software as part of a Desktop, Mobile, or Web Application is permitted in object code form only.
```

Comment:
This new term - while I agree with it - justifies IMO giving this license a version 1.3.

Reply:
Why does it prevent you from distributing the application source code? The “Software” refers to Slint here, not the application source code.

Reply:
This indeed forbids vendoring Slint, right? Is using git submodules for Slint then allowed? If so, how is that different from vendoring? If not so then how is using git submodules different just pulling Slint through the package manager?
Also you can still distribute patches to slint, and they can be applied automatically by the build system.
The clause is very weird.

Reply:
Distributing a git repository where one of the files contained within (`.gitmodules`) contains the url to the slint repo does IMO not constitute distribution of the Slint sources (merely a url to it).

I think the scenario to address here is that somebody decides to license Slint under the terms of the royalty free license, wraps it (Slint; the “Software”) into their own SDK (let’s call it Flint) and sells Flint as GUI toolkit for Rust and C++ developers.

@aurindam is that accurate in terms of intent?

Reply:
Yes, I think we have the same understanding.. that is, to **explicitly** prevent the scenarios like below:

> I think the scenario to address here is that somebody decides to license Slint under the terms of the royalty free license, wraps it (Slint; the “Software”) into their own SDK (let’s call it Flint) and sells Flint as GUI toolkit for Rust and C++ developers.

and 

> I mean I kinda get the point why it was added, as you can work around distributing Slint by itself by just making a very dumb application right now.


Maybe a simpler way to phrase it would be - "You are not allowed to resell Slint" instead of restricting distribution to object code since thats the root issue.